packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -153,6 +155,7 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
     alt = 'Thumbnail',
     onClick = null,
     focusPoint = DEFAULT_PROPS.focusPoint,
+    imgProps,
```

Comment:
It would be more consistent to set fallback with empty object here?
`imgProps = {}`

Reply:
Unfortunately, that will trigger a snapshot update on all the components that use `Thumbnail` on the product side, thus making this change a breaking one